src/pages/index.js

```diff
@@ -33,7 +33,10 @@ import {
   settings,
   editForm,
   addForm,
+  editProfilePicButton,
 } from "../utils/constants.js";
+import Api from "../components/Api.js";
+import PopupWithConfirmation from "../components/PopupWithConfirmation";
```

Comment:
Needs correction: when I open the profile form the value in the description input is `undefined`